src/service/follower.proto

```diff
@@ -46,6 +38,23 @@ message follower_gateway_resp_v1 {
   region region = 7;
 }
 
+message gateway_not_found {}
```

Comment:
might be better to include a generic error response which can cater to any future error handling ( including any additional APIs which get added over time)...something like:

enum follower_error {
  gateway_not_found = 0;
}

message follower_error_resp {
  follower_error error = 1;  
  bytes details = 2;
}

In the case of a gateway not found, the error would equal the gateway_not_found enum value, whilst details would be the pub key specified in the request

Reply:
done [54c1367](https://github.com/helium/proto/pull/209/commits/54c136709e9570979757fe2d1baf972bf98aa15a)